### ISSUE TITLE
Implement TORQUE options in OpenPBSDriver

### ISSUE THREAD

**Issue by pinkwah** [equinor/ert#7101]
The following QUEUE_OPTION TORQUEs:
1. #7222 
2. `NUM_NODES`, `NUM_CPUS_PER_NODE`
3. `KEEP_QSUB_OUTPUT`. First, check whether output of `job_dispatch.py` is logged anywhere. If it is, then don't implement this option.

**Comment by jonathan-eq** [comment]
BLOCKED BY TORQUE BUGS #7143 #7138

**Comment by xjules** [comment]
> BLOCKED BY TORQUE BUGS #7143 #7138

Fixed now, so unblocking.

**Comment by jonathan-eq** [comment]
Should we implement QUEUE_QUERY_TIMEOUT for qstat? I think we poll often enough to mitigate the flakiness anyways.

**Comment by berland** [comment]
> Should we implement QUEUE_QUERY_TIMEOUT for qstat? I think we poll often enough to mitigate the flakiness anyways.

No. Agree.

**Comment by berland** [comment]
Closing in favour of specific issues #7222, #7232, #7233
